content/en/blog/2025/ottl-contexts-just-got-easier.md

```diff
@@ -0,0 +1,153 @@
+---
+title: OTTL contexts just got easier with context inference
+linkTitle: OTTL contexts just got easier
+date: 2025-02-17
+author: '[Edmo Vamerlatti Costa](https://github.com/edmocosta) (Elastic)'
+draft: true # TODO: remove this line once your post is ready to be published
+issue: 6289
+sig: Collector SIG
+cSpell:ignore: OTTL Vamerlatti
+---
+
+Selecting the right OTTL context for running statements can be challenging, even
+for experienced users. Choosing the correct option impacts both accuracy and
+efficiency, as using higher-level OTTL contexts can avoid unnecessary iterations
+through nested lower-level contexts.
+
+To simplify this process, the OpenTelemetry community is excited to announce
+OTTL
+[context inference](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/transformprocessor/README.md#context-inference)
+support for the
+[transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/transformprocessor).
+This feature removes the need to manually specify OTTL contexts, improving
+statement processing efficiency by automatically selecting the most appropriate
+one. This optimization ensures that data transformations are both accurate and
+performant, allowing users to focus solely on their data without needing to
+understand the underlying concept of OTTL contexts.
+
+## How does it work?
+
+Starting with version `0.120.0`, the transform processor supports two new
+[context-inferred configuration](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/transformprocessor/README.md#context-inferred-configurations)
+styles. The first one offers a simpler and flatter approach, while the second
+closely resembles the existing configuration format.
```

Comment:
I'd like to get https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37909 merged and then we can update these links to point to the 2 new configuration sections.

Reply:
merged, ready to update

Reply:
Done, configuration names and links are now updated.